portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+jQuery(document).ready(function($) {
+
+
+	var mastheadheight = $('.ds-header').outerHeight();
+    console.log(mastheadheight);
+	$(".ds-banner,.ds-main-section").css("margin-top" , mastheadheight);
+
+	$(window).scroll(function(){
+	    if ($(window).scrollTop() >= 10) {
+	        $('.ds-header').addClass('ds-fixed-header');
+	    }
+	    else {
+	        $('.ds-header').removeClass('ds-fixed-header');
+	    }
+	}).scroll();
+
+
+ });
+
+ /**
+  * fetches the string from servlet
+  */
+
+  async function getString(){
+      const responseFromServer= await fetch('/hello');
+      const textFromResponse= await responseFromServer.json();
```

Comment:
I'd recommend running a linter to format everything, `textFromResponse` should have a space right after.

Reply:
is there a way to add it as a plugin in the shell?

Reply:
I think the easiest way is to format in the Cloud Editor, like this: ![image](https://user-images.githubusercontent.com/76013657/173127420-107b876d-a2a9-4d85-8973-8edbbf76d1a2.png).

There's some settings you can change regarding formatting in the settings, if that's not enough you could try installing ESLint manually from the command line: https://eslint.org/docs/user-guide/getting-started.